app/src/actions/FirebaseActions.js

```diff
@@ -638,3 +640,17 @@ export async function signOut() {
     trackLogoutFailed();
   }
 }
+
+export async function updateUserInFirebaseAuthUser(data) {
```

Comment:
add a comment here telling the type of data, since we don't have TS this should be made clear separately